Add semantic html content (#3450)
Codecademy/docs: content/html/concepts/semantic-html/semantic-html.md

```html
<!-- Semantic HTML -->
<footer>
  <p>This is a footer.</p>
</footer>
```

> ![Semantic Tags](https://raw.githubusercontent.com/Codecademy/docs/main/media/<semantic-tags>.<png>)

Reviewer: ```suggestion
![Semantic Tags](https://raw.githubusercontent.com/Codecademy/docs/main/media/semantic-tags.png)
```
Ad a sentence before this image describing what it shows.

## Why Use Semantic HTML?

- Accessibility: Semantic HTML makes webpages accessible for mobile devices and for people with disabilities as well. This is because screen readers and browsers are better able to interpret the content of the site.

- Readability: Semantic HTML also makes the website’s source code easier to read for other web developers.

- SEO: It improves the website SEO, or Search Engine Optimization, which is the process of increasing the number of people that visit a webpage. With better SEO, search engines are better able to identify the content of a website and weigh the most important content appropriately.

## Testing and Validating for Browser Compatibility

> Testing and validating for browser compatibility is essential in web development to ensure a consistent user experience. Stick to well-supported modern semantic HTML tags, checking their compatibility using resources like [Can I use](https://caniuse.com/). Additionally, employ validation tools like the [W3C Markup Validation Service](https://validator.w3.org/) to maintain code quality, identify errors, and ensure compliance with web standards, enhancing compatibility across various browsers and devices.

Reviewer 2: ```suggestion
Testing and validating for browser compatibility is essential in web development to ensure a consistent user experience. It is best practice to employ well-supported modern semantic HTML tags, and confirming their compatibility using resources like [Can I use](https://caniuse.com/). Additionally, validation tools like the [W3C Markup Validation Service](https://validator.w3.org/) are valuable for maintaining code quality, identifying errors, and ensuring compliance with web standards. These practices can aid in supporting compatibility across various browsers and devices.
```

## Semantic HTML Best Practices

> Use Semantics for Meaning: Choose semantic HTML Tags wisely. It is important to choose tags that accurately convey the content's meaning and purpose. Avoid common mistakes like using `<article>` and `<section>` tags interchangeably. In the following example, using `<article>` for each subsection correctly represents self-contained content (news articles), while `<section>` encapsulates the broader category 'Technology News,' aligning with their intended semantic purposes.

Reviewer 2: ```suggestion
It is important to choose tags that accurately convey the content's meaning and purpose. Avoid common mistakes like using `<article>` and `<section>` tags interchangeably. In the following example, using `<article>` for each subsection correctly represents self-contained content (news articles), while `<section>` encapsulates the broader category 'Technology News,' aligning with their intended semantic purposes.
```

> Use Semantics for Meaning: Choose semantic HTML Tags wisely: It is important to choose tags that accurately convey the content's meaning and purpose. Avoid common mistakes like using `<article>` and `<section>` tags interchangeably. In the following example, using `<article>` for each subsection correctly represents self-contained content (news articles), while `<section>` encapsulates the broader category 'Technology News,' aligning with their intended semantic purposes.

Reviewer: ```suggestion
Use Semantics for Meaning: Choose semantic HTML Tags wisely. It is important to choose tags that accurately convey the content's meaning and purpose. Avoid common mistakes like using `<article>` and `<section>` tags interchangeably. In the following example, using `<article>` for each subsection correctly represents self-contained content (news articles), while `<section>` encapsulates the broader category 'Technology News,' aligning with their intended semantic purposes.
```

```html
<section>
  <h1>Technology News</h1>
  <article>
    <h2>Latest Smartphone Releases</h2>
    <p>Discover the newest smartphones hitting the market...</p>
    <!-- Article content here -->
  </article>
  <article>
    <h2>Upcoming Tech Events</h2>
    <p>Stay updated on the latest tech events & conferences...</p>
    <!-- Article content here -->
  </article>
</section>
```

> ![Semantic Tag Visualization for Technology News](https://raw.githubusercontent.com/Codecademy/docs/main/media/<use-semantics-for-meaning>.<png>)

Reviewer: ```suggestion
![Semantic Tag Visualization for Technology News](https://raw.githubusercontent.com/Codecademy/docs/main/media/use-semantics-for-meaning.png)
```

> Proper Nesting: Proper nesting is a fundamental aspect of semantic HTML. So, it is crucial to structure content logically to reflect hierarchical relationships between semantic elements. For instance, avoid the common mistake of placing the `<aside>` element directly within the main content; `<aside>` is intended to be positioned to the side, separate from the primary content.

Reviewer 2: ```suggestion
Proper nesting is a fundamental aspect of semantic HTML. So, it is crucial to structure content logically to reflect hierarchical relationships between semantic elements. For instance, avoid the common mistake of placing the `<aside>` element directly within the main content. The `<aside>` tag is intended for content that is to be positioned to the side, or flanking, separate from the primary content.
```

> Proper Nesting: Structure with intent. Proper nesting is a fundamental aspect of semantic HTML. So, it is crucial to structure content logically to reflect hierarchical relationships between semantic elements. For instance, avoid the common mistake of placing the `<aside>` element directly within the main content; `<aside>` is intended to be positioned to the side, separate from the primary content.

Reviewer: ```suggestion
Proper Nesting: Proper nesting is a fundamental aspect of semantic HTML. So, it is crucial to structure content logically to reflect hierarchical relationships between semantic elements. For instance, avoid the common mistake of placing the `<aside>` element directly within the main content; `<aside>` is intended to be positioned to the side, separate from the primary content.
```

- Readability: Semantic HTML also makes the website’s source code easier to read for other web developers.

- SEO: It improves the website SEO, or Search Engine Optimization, which is the process of increasing the number of people that visit a webpage. With better SEO, search engines are better able to identify the content of a website and weigh the most important content appropriately.

## Testing and Validating for Browser Compatibility

Testing and validating for browser compatibility is essential in web development to ensure a consistent user experience. Stick to well-supported modern semantic HTML tags, checking their compatibility using resources like [Can I use](https://caniuse.com/). Additionally, employ validation tools like the [W3C Markup Validation Service](https://validator.w3.org/) to maintain code quality, identify errors, and ensure compliance with web standards, enhancing compatibility across various browsers and devices. 

> ## Semantic HTML Best Practices: Mastering the Art

Reviewer: ```suggestion
## Semantic HTML Best Practices
```

> - Use Semantics for Meaning: Choose semantic HTML Tags wisely: It is important to choose tags that accurately convey the content's meaning and purpose. Avoid common mistakes like using `<article>` and `<section>` tags interchangeably. In the following example, using `<article>` for each subsection correctly represents self-contained content (news articles), while `<section>` encapsulates the broader category 'Technology News,' aligning with their intended semantic purposes.

Reviewer: ```suggestion
Use Semantics for Meaning: Choose semantic HTML Tags wisely. It is important to choose tags that accurately convey the content's meaning and purpose. Avoid common mistakes like using `<article>` and `<section>` tags interchangeably. In the following example, using `<article>` for each subsection correctly represents self-contained content (news articles), while `<section>` encapsulates the broader category 'Technology News,' aligning with their intended semantic purposes.
```

```html
<section>
  <h1>Technology News</h1>
  <article>
    <h2>Latest Smartphone Releases</h2>
    <p>Discover the newest smartphones hitting the market...</p>
    <!-- Article content here -->
  </article>
  <article>
    <h2>Upcoming Tech Events</h2>
    <p>Stay updated on the latest tech events and conferences...</p>
    <!-- Article content here -->
  </article>
</section>
```

- Proper Nesting: Structure with intent. Proper nesting is a fundamental aspect of semantic HTML. So, it is crucial to structure content logically to reflect hierarchical relationships between semantic elements. For instance, avoid the common mistake of placing the `<aside>` element directly within the main content; `<aside>` is intended to be positioned to the side, separate from the primary content.

```html
<main>
  <article>
    <!-- Main content -->
  </article>
  <aside>
    <!-- Sidebar content -->
  </aside>
</main>
```

> ## Drawbacks of Using Semantic HTML: Navigating Challenges

Reviewer: ```suggestion
## Drawbacks of Using Semantic HTML
```
